scopes/workspace/workspace/workspace.graphql.ts

```diff
@@ -59,7 +60,7 @@ export default (workspace: Workspace, graphql: GraphqlMain) => {
         name: String
         path: String
         icon: String
-        components(offset: Int, limit: Int): [Component]
+        components(ids: [String!], offset: Int, limit: Int): [Component]
```

Comment:
I think all we need to do is add the activeLane as a property? then use useLanes for the specific gallery pages once clicked the switcher.

Reply:
Lets discuss this further, I am not sure we need to pass active lane to GQL
If we are planning to use useLanes then shouldnt the workspace/scope root be aware of the active lane in the UI instead?